share/completions/flatpak.fish

```diff
@@ -30,9 +31,21 @@ complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-i
 complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-sign -d 'Sign an application or runtime'
 complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-update-repo -d 'Update the summary file in a repository'
 
-complete -f -c flatpak -n "__fish_seen_subcommand_from run" -a "(flatpak list --app | string match -r '\S+')"
-complete -f -c flatpak -n "__fish_seen_subcommand_from info uninstall" -a "(flatpak list | string match -r '\S+')"
-complete -f -c flatpak -n "__fish_seen_subcommand_from remote-ls remote-modify remote-delete" -a "(flatpak remote-list | string match -r '\S+')"
+if test (string join . "$flatversion[1]" "$flatversion[2]") -ge 1.2
+    for line in (flatpak list --app --columns=application,name)
+        set -l info (string split \t "$line")
+        complete -f -c flatpak -n "__fish_seen_subcommand_from run" -a "$info[1]" -d "$info[2]"
```

Comment:
This `string split` is unnecessary, because bits after a tab are already used as a description for the line.

So, in total, this entire for-loop can just be:

```fish
complete -f -c flatpak -n "__fish_seen_subcommand_from run" -a '(flatpak list --app --columns=application,name)'
```